app/services/metrc_service/package/start.rb

```diff
@@ -54,34 +54,37 @@ def create_package_payload
           RemediationDate: nil,
           RemediationSteps: nil,
           ActualDate: package_date,
-          Ingredients: start_consume_completions.map do |consume|
+          Ingredients: consume_completions.map do |consume|
             harvest_ingredient(consume)
           end
         }]
       end
 
       def testing?
-        seeding_unit.name == 'Testing Package'
+        batch.arbitrary_id =~ /test/i
       end
 
       def tag
         batch.relationships.dig('barcodes', 'data', 0, 'id')
       end
 
       def item_type
-        resource_unit_name = resources.first.resource_unit.name
+        # resource_unit_name = resource_units.name
 
-        # try the format: [unit] of [type] - [strain]
-        matches = resource_unit_name.match(/^[\w]+ of ([\w\s]+) - [\w\s]+$/)
+        # # try the format: [unit] of [type], [strain]
+        # type = resource_unit_name[/^[\w]+ of ([\w\s]+), [\w\s]+$/]
 
-        # then try the format: [type] - [strain]
-        matches = resource_unit_name.match(/^([^\-]+) - [\w\s]+$/) if matches.nil?
+        # # then try the format: [type], [strain]
+        # type = resource_unit_name[/^([^\-]+), [\w\s]+$/] if type.nil?
 
-        return matches[1] unless matches.nil?
+        # raise InvalidAttributes,
+        #       "Item type could not be extracted from the resource unit name: #{resource_unit_name}. " \
+        #       "Expected the format '[unit] of [type], [strain]' or '[type], [strain]'" \
+        #       if type.nil?
+        #
+        # type
```

Comment:
Please delete the commented code if we no longer need it.

Reply:
We may possibly need this. Hardcoding the only accepted option for Metrc `Flower` for now, but this block come back to life.